packages/apps/transfer/.eslintrc.js

```diff
@@ -1,8 +1,32 @@
+const fs = require('fs');
+
+const prettierOptions = JSON.parse(fs.readFileSync('./.prettierrc', 'utf8'));
 require('@rushstack/eslint-config/patch/modern-module-resolution');
 
 module.exports = {
-  plugins: ['jest'],
+  plugins: ['prettier', 'jest'],
   extends: ['next/core-web-vitals', '@kadena-dev/eslint-config/profile/react'],
   parserOptions: { tsconfigRootDir: __dirname },
   ignorePatterns: ['**/*.spec.ts', '**/*.spec.tsx', '**/generated/**'],
+  rules: {
+    'prettier/prettier': ['error', prettierOptions],
+    '@typescript-eslint/strict-boolean-expressions': 0,
+    'arrow-body-style': [2, 'as-needed'],
+    'comma-dangle': [2, 'always-multiline'],
+    'prefer-template': 2,
+    'import/newline-after-import': 1,
+    'import/no-unresolved': 2,
```

Comment:
this is super useful especially for windows users with a repo manager
since we have a rush, on windows we'll have root node modules, therefore if a package is not listed in the package.json of one of the packages, it'll allow you to use it since you have it in root, but then on deployment it figures it out and it.
but with this rule, if you import a package, it has to be listed in package.json

this issue happens often on windows machines